yellowbrick/text/postag.py

```diff
@@ -150,70 +169,34 @@ def fit(self, X, y=None, **kwargs):
             self._handle_universal(X,y)
         self.draw()
         return self
-
+    
     def _penn_tag_map(self):
         """
         Returns a Penn Treebank part-of-speech tag map.
         """
-        
-        default_dic =  {
-                            "noun": 0,
-                            "verb": 0,
-                            "adjective": 0,
-                            "adverb": 0,
-                            "preposition": 0,
-                            "determiner": 0,
-                            "pronoun": 0,
-                            "conjunction": 0,
-                            "infinitive": 0,
-                            "wh- word": 0,
-                            "modal": 0,
-                            "possessive": 0,
-                            "existential": 0,
-                            "punctuation": 0,
-                            "digit": 0,
-                            "non-English": 0,
-                            "interjection": 0,
-                            "list": 0,
-                            "symbol": 0,
-                            "other": 0,
-                        }
-        if self.stack:
-            penn_tag_map={}
-            for i in self.labels_:
-                penn_tag_map[i]=default_dic.copy()
-            return penn_tag_map
-        else:
-            return default_dic
-
+        self.pos_tags = PENN_TAGS
+        return self._make_tag_map(PENN_TAGS)
+    
     def _uni_tag_map(self):
         """
         Returns a Universal Dependencies part-of-speech tag map.
         """
-        default_dic = {
-                        "noun": 0,
-                        "verb": 0,
-                        "adjective": 0,
-                        "adverb": 0,
-                        "adposition": 0,
-                        "determiner": 0,
-                        "pronoun": 0,
-                        "conjunction": 0,
-                        "infinitive": 0,
-                        "punctuation": 0,
-                        "number": 0,
-                        "interjection": 0,
-                        "symbol": 0,
-                        "other": 0,
-                    }
-        if self.stack:
-            uni_tag_map={}
-            for i in self.labels_:
-                uni_tag_map[i]=default_dic.copy()
-            return uni_tag_map
-        else:
-            return default_dic
+        self.pos_tags = UNIVERSAL_TAGS
```

Comment:
Is it necessary to store `pos_tags` on the visualizer? (e.g. are you using this property elsewhere?) This seems like a bit of a duplicate with `self.tagset` and the keys on the `pos_tag_counts_`.

Reply:
Yup! It is duplicate but I used it at certain places to de-duplicate the code. Like in `finalize()` it leads to common xticks for both `frequency=True` and `frequency=False`. It is not used extensively and can be removed if you suggest!!

Reply:
No worries - if you're using it to deduplicate code then that's fine with me! My preference, however, would be to make it `self._pos_tags` -- the sklearn API suggests the following for estimator attributes:

- no underscore affix (`self.foo`): a hyperparameter passed in by the user in `__init__`
- underscore suffix  (`self.foo_`): a learned parameter that only exists on the class after calling `fit()` 
- underscore prefix (`self._foo`): any other required class property that is not a method

Reply:
Thanks for providing me with regulations!